codenav/src/utils.c

```diff
@@ -19,6 +19,8 @@
  *  along with this program; if not, see <http://www.gnu.org/licenses/>.
  */
 
+#include <geanyplugin.h>
+
 #include "utils.h"
 
```

Comment:
As an alternative one could replace the inappropriate use of utils_str_equal with g_strcmp/0 but i dont like to potentially change the behavior now